hathor/indexes/memory_address_index.py

```diff
@@ -30,11 +30,20 @@
 class MemoryAddressIndex(AddressIndex):
     """ Index of inputs/outputs by address
     """
+
+    index: DefaultDict[str, Set[bytes]]
+
     def __init__(self, pubsub: Optional['PubSubManager'] = None) -> None:
-        self.index: DefaultDict[str, Set[bytes]] = defaultdict(set)
         self.pubsub = pubsub
         if self.pubsub:
             self.subscribe_pubsub_events()
+        self.force_clear()
```

Comment:
I guess we should call this method at the beginning of `__init__`. I know it's unliked that this index will be created after the full node has already been initialized. So pubsub would be empty anyway. But the call to `subscribe_pubsub_event()` might trigger events right away, which could cause an exception.

Reply:
OK, that makes sense.

Reply:
Done.

Reply:
Thanks!